docs/content/preview/troubleshoot/nodes/disk-full.md

```diff
@@ -1,25 +1,136 @@
+
+When dropping YSQL tables, ensure the YB-Master has enough disk space, as this involves updating the YSQL catalog tables.
+
+### Disable database features that require more disk space
+
+The following features require more disk space. You can disable or turn these features off to reclaim space on the nodes.
+
+- Time-to-live (TTL)
+- Point-in-time recovery (PITR)
+- xCluster
+- Change Data Capture (CDC)
+
+### Run manual compaction
+
+YugabyteDB automatically compacts data to keep the database running efficiently and reduce disk use. Certain operations, like bulk load of data or deleting a large number of rows, can cause a temporary spike in disk usage. If waiting for the automatic compaction task is not an option, you can manually run the compaction task using the following commands:
+
+- [Compact individual tables](../../../admin/yb-admin/#compact-table)
+- [Compact individual tablets](../../../admin/yb-ts-cli/#compact-tablet)
+
+Note that regardless of the above options, it is possible that some other activity on the node (such as remote bootstrap) can end up taking up more space and use up the space on the disk.
+
+The following sections describe the disk full scenarios and the recommended recovery actions.
+
+## Scenarios
+
+### The drive containing the log file is full
+
+During the tablet bootstrap that takes place during a YB-TServer restart, a new WAL file is allocated, irrespective of the amount of logs generated in the server's previous WAL file. If a YB-TServer has several tablets and results in a crash loop, each one of the tablets creates new WAL files on every bootstrap, filling up the disk. The process is unable to create new log files at tablet bootstrap time, and this can be identified by looking at the `/var/log/messages` files.
+
+### Disk full situation on other data drives
+
+#### Crash loop due to write I/O error
+
+If the disk isn't completely full (it probably has a few KBs of disk space left), the YB-TServer can successfully bootstrap. However, a subsequent write operation might fail with a fatal/crash error due to the disk getting full. This can lead to a continuous crash loop as the server restarts and encounters the same issue all over again.
+An example IO error is as follows:
+
+```output
+consensus_queue.cc:368] Check failed: _s.ok() Bad status: IO error (yb/util/env_posix.cc:504): /mnt/d1/yb-data/tserver/wals/table-6e527d21c179455286ae94ddcbebd267/tablet-be46450d80d049ee8b6a0780dfa54926/.tmp.newsegmentiS4vyN: No space left on device (system error 28)
+```
+
+#### Automatic YB-TServer blacklisting
+
+During YB-TServer startup, the FSManager (File System Manager) module checks the health of each data drive by creating a small test file. If the disk is completely full, the YB-TServer does the following:
+
+1. Skips the full drive: The test file creation fails, the node ignores this particular data drive, and marks it faulty. It also increments the `drive_fault` metric.
```

Comment:
Fix the numbering - 1, 2, 3

Reply:
Oh it is correct currently. It is a markdown standard. The list will render as 1,2,3 finally.